### PR TITLE
added update metadata with parametrized seller fee

### PR DESCRIPTION
**Pull request by halaprix**
Added the ability to update metadata and set the seller fee (was set to 0 by default)

```
update_token_metadata(api_endpoint, source_account, mint_token_id, link, data, creators_addresses, creators_verified, creators_share, fee)
```

### CODE REVIEW

**Comment by jarry-xiao** [comment]
Thanks looks good

**Comment by nshunt639** [comment]
Is it possible to update the metadata before minting?

**Comment by crypt0miester** [comment]
> Is it possible to update the metadata before minting?

which metadata? this pr does exactly update even after minting.

File: metaplex/metadata.py [hunk 1]
```diff
@@ -181,8 +182,8 @@ def get_metadata(client, mint_key):
     metadata = unpack_metadata_account(data)
     return metadata
 
-def update_metadata_instruction_data(name, symbol, uri, creators, verified, share):
-    _data = bytes([1]) + _get_data_buffer(name, symbol, uri, creators, verified, share) + bytes([0, 0])
+def update_metadata_instruction_data(name, symbol, uri, fee, creators, verified, share):
+    _data = bytes([1]) + _get_data_buffer(name, symbol, uri, fee, creators,  verified, share) + bytes([0, 0])
```
Comment (crypt0miester) [inline metaplex/metadata.py:186]: @jarry-xiao 

Mr. Jarry, is the byte at the "+ bytes([0, 0])" for primary_sale_happened and is_mutable? 

I think they are the ones we would change if we would like to change the primary_sale_happened and is_mutable right?

**Pull request merged**